spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/session/RedisSessionProperties.java

```diff
@@ -40,6 +40,11 @@
 	 */
 	private RedisFlushMode flushMode = RedisFlushMode.ON_SAVE;
 
+	/**
+	 * Allows specifying a strategy for configuring and validating Redis.
+	 */
+	private RedisAction action = RedisAction.NOTIFY_KEYSPACE_EVENTS;
```

Comment:
The action interface is quite generic and allows the user to do anything they want to configure and validate the connection. I don't think "action" is the right term for this as what we set is a "configuration and validation strategy". 

`configurationStrategy` is a bit mouthful but more explicit perhaps?

Reply:
Having in mind the purpose of `ConfigureRedisAction`, naming this property `configurationStrategy` would be more appropriate.